### PR TITLE
Update Explorer links on Tangle Dapp + getUrlExplorer + Chains Config

### LINKED ISSUE #2224: [TASK] Update getExplorerURI function for substrate transaction and address + Making sure all tx links on Tangle Dapp are correct
### Description 

On PolkadotJs Dashboard there is no way for the users to query for the info of a specific tx or address. We need to use update the `getExplorerURI` as well as the polkadotExplorerUrl for Tangle Mainnet & Testnet to use Statescan
- Mainnet: https://tangle.statescan.io/#/
- Testnet: https://tangle-testnet.statescan.io/

```
export const getExplorerURI = (
  explorerUri: string,
  addOrTxHash: string,
  variant: 'tx' | 'address',
  txProviderType: WebbProviderType
): URL => {
  switch (txProviderType) {
    case 'web3':
      return new URL(`${variant}/${addOrTxHash}`, explorerUri);

    case 'polkadot': {
      const path = variant === 'tx' ? `#/explorer/query/${addOrTxHash}` : '';
      return new URL(`${path}`, explorerUri);
    }

    default:
      return new URL('');
  }
};
```

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes

_Provide a detailed description of proposed changes._

- Navigate users to correct tx and account page on block explorer (both EVM & Substrate)
- Update chains-config in `dapp-config`
- Update `getUrlExplorer` function that handle substrate case successfully
- Unify constants related to `Tangle` inside `dapp-config`

### Proposed area of change

_Put an `x` in the boxes that apply._

- [x] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [x] `apps/tangle-dapp`
- [ ] `apps/testnet-leaderboard`
- [ ] `apps/faucet`
- [ ] `apps/zk-explorer`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. `Closes #233`)._

- Closes #2224

### Screen Recording

_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/5594d467-75ff-4b28-8842-1ada33e9a2b6

---

### Code Checklist

_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/faucet/src/constants/sidebar.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import {
+  TANGLE_TESTNET_EVM_EXPLORER_URL,
+  TANGLE_TESTNET_NATIVE_EXPLORER_URL,
+} from '@webb-tools/dapp-config/constants/tangle';
```
Comment (AtelyPham) [inline apps/faucet/src/constants/sidebar.ts:4]: Same here.

File: apps/testnet-leaderboard/components/Navbar.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use client';
 
 import { AccordionTrigger } from '@radix-ui/react-accordion';
+import { TANGLE_TESTNET_NATIVE_EXPLORER_URL } from '@webb-tools/dapp-config/constants/tangle';
```
Comment (AtelyPham) [inline apps/testnet-leaderboard/components/Navbar.tsx:4]: Same here.

File: apps/bridge-dapp/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import {
   useWebContext,
   useConnectWallet,
 } from '@webb-tools/api-provider-environment';
+import { TANGLE_TESTNET_NATIVE_EXPLORER_URL } from '@webb-tools/dapp-config/constants/tangle';
```
[the remark this answers is not in the document]
Comment (AtelyPham) [reply apps/bridge-dapp/src/components/Header/Header.tsx:5]: My bad.

**Review by AtelyPham — APPROVED** [review]

**Review by monaiuu — APPROVED** [review]

**Review by monaiuu — COMMENTED** [review]
@vutuanlinh2k2 should we enable the explorer to open without wallet connection from `view explorer` link to polkadotJS portal? I was unable to do so in the deployment preview link.

**Comment by vutuanlinh2k2** [comment]
> @vutuanlinh2k2 should we enable the explorer to open without wallet connection from `view explorer` link to polkadotJS portal? I was unable to do so in the deployment preview link.

Hmm how about I let the user open the Block Explorer's home page instead of their account page? I believe those are more user-friendly than the polkadotjs portal

**Pull request merged**